basyx.common/basyx.filerepository-backend-mongodb/src/main/java/org/eclipse/digitaltwin/basyx/core/filerepository/MongoDBFileRepository.java

```diff
@@ -89,38 +101,38 @@ public void delete(String fileId) throws FileDoesNotExistException {
 
 		if (!exists(fileId))
 			throw new FileDoesNotExistException();
-		
+
 		String mongoDBfileId = getFileId(fileId);
 
```

Comment:
What would happen if getFileId returns an empty string "". Will this cause an exception of the deltete operation of MongoDb?